system/camerad/sensors/os04c10.cc

```diff
@@ -20,6 +20,17 @@ const uint32_t os04c10_analog_gains_reg[] = {
 
 }  // namespace
 
+void OS04C10::OS04C10_IFE_ds_override() {
```

Comment:
don't put the class name in the function again. also ds is downscale? maybe call this `ife_downscale_configure`?